lib/code_sponsor_web/controllers/coherence/session_controller.ex

```diff
@@ -0,0 +1,371 @@
+defmodule CodeSponsorWeb.Coherence.SessionController do
+  @moduledoc """
+  Handle the authentication actions.
+
+  """
+  use Coherence.Web, :controller
+  use Timex
+  use Coherence.Config
+
+  import Coherence.TrackableService
+  import Ecto.Query
+  import Coherence.Schemas, only: [schema: 1]
+  # import Coherence.Rememberable, only: [hash: 1, gen_cookie: 3]
```

Comment:
I think we need this don't we?

Reply:
I don't think so, this is all exactly what we're using now as its a direct copy from the library. Tested it locally and looks like it still runs the Rememberable queries.